packages/opa-react/tests/authz.test.tsx

```diff
@@ -1,4 +1,5 @@
 import React from "react";
+import "@testing-library/jest-dom/vitest"; // TODO(sr): why do we need this here? (it's in setup.ts already)
```

Comment:
With this addition, the types of the extra matchers are resolved: [this line below](https://github.com/StyraInc/opa-typescript/pull/75/files#diff-699c6da43fa195b0186138e07dc1e03e2bb505974ba99367c0e76fc52762fe23R51) gets no red squiggly underline.

**But** I've been thinking that there _must_ be a better way?! I thought doing the right thing to `tests/setup.ts` would help, but I seem to be unable to get it to work 😑